src/main/java/io/kestra/plugin/ldap/Search.java

```diff
@@ -99,7 +99,6 @@ public class Search extends LdapConnection implements RunnableTask<Search.Output
                 `--> This special attribute canno't be combined with other attributes and the search will ignore everything else.
             """
     )
-    @PluginProperty(dynamic = true)
```

Comment:
You should not remove the annotation but remove the dynamic attribute instead so this is correctly documented as not-dynamic.

Reply:
Changement has been made.